### PR TITLE
Constantinc 2830 validate project process extensions

### PR DESCRIPTION
**Pull request by constantin-ciobotaru**

### CODE REVIEW

**Review by pancudaniel7 — COMMENTED** [review]
2 small comments

File: activiti-cloud-services-org/activiti-cloud-services-org-service/src/main/java/org/activiti/cloud/services/organization/validation/ExtensionsModelValidator.java [hunk 1]
```diff
@@ -15,26 +15,231 @@
...
+    private Optional<Model> findModelInContext(String modelId,
+                                               ValidationContext validationContext) {
+        return validationContext.getAvailableModels()
+                .stream()
+                .filter(model -> Objects.equals(model.getId(),
+                                                modelId))
+                .findFirst();
+    }
+
+    private Model convertBytesToModel(byte[] bytes) {
```
Comment (pancudaniel7) [inline activiti-cloud-services-org/activiti-cloud-services-org-service/src/main/java/org/activiti/cloud/services/organization/validation/ExtensionsModelValidator.java:235]: This method should not be moved to ProcessModelConverter class ?

File: activiti-cloud-services-org/activiti-cloud-services-org-service/src/main/java/org/activiti/cloud/services/organization/validation/ExtensionsModelValidator.java [hunk 2]
```diff
@@ -15,26 +15,231 @@
...
+        try {
+            return extensionsConverter.convertToEntity(bytes);
+        } catch (ModelingException ex) {
+            throw new SyntacticModelValidationException("Cannot convert json extensions to a model",
+                                                        ex);
+        }
+    }
+
+    private Optional<BpmnProcessModelContent> convertToBpmnModel(byte[] bytes) {
```
Comment (pancudaniel7) [inline activiti-cloud-services-org/activiti-cloud-services-org-service/src/main/java/org/activiti/cloud/services/organization/validation/ExtensionsModelValidator.java:244]: Also this method should not be moved to ProcessModelConverter class ?

File: activiti-cloud-services-org/activiti-cloud-services-org-service/src/main/java/org/activiti/cloud/services/organization/validation/ExtensionsModelValidator.java (hunk 1 above)
Comment (constantin-ciobotaru) [reply activiti-cloud-services-org/activiti-cloud-services-org-service/src/main/java/org/activiti/cloud/services/organization/validation/ExtensionsModelValidator.java:235]: This is just to cover the Exception to a proper one.

**Review by erdemedeiros — COMMENTED** [review]
@constantin-ciobotaru can you check warnings from Codacy, please? Otherwise, it looks good to me.

**Comment by constantin-ciobotaru** [comment]
@erdemedeiros "can you check warnings from Codacy, please?" -> Done. Also rebased.

**Pull request merged**